### PR TITLE
authentication

### PR DESCRIPTION
**Pull request by saiganesharaja**
Screenshot from 2024-02-21 23-49-57
Screenshot from 2024-02-21 23-50-28

### CODE REVIEW

**Comment by saiganesharaja** [comment]
Screenshot from 2024-02-23 12-26-42

**Comment by saiganesharaja** [comment]
Screenshot from 2024-02-23 12-33-37

**Comment by saiganesharaja** [comment]
Screenshot from 2024-02-23 16-02-06

**Comment by saiganesharaja** [comment]
Screenshot from 2024-02-23 16-02-56

**Comment by saiganesharaja** [comment]
image

**Comment by saiganesharaja** [comment]
Screenshot from 2024-02-27 15-15-33
Screenshot from 2024-02-27 15-15-18

**Comment by saiganesharaja** [comment]
Screenshot from 2024-02-27 15-20-27

File: src/app/signup/page.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
+'use client';
+import React, { useEffect, useState } from 'react';
+import { EnvironmentOutlined, HeatMapOutlined, LockOutlined, PhoneOutlined, UserOutlined, YuqueOutlined } from '@ant-design/icons';
+import { Button, Checkbox, Flex, Form, Input, Typography } from 'antd';
+
+
+const App: React.FC = () => {
```
Comment (Namahsivaya7) [inline src/app/signup/page.tsx]: change the name insted of app

File: src/app/signin/page.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    <center style={{padding:'100px 400px'}}>
+    <Form 
+    
+    style={{height:'400px',backgroundColor:'rgba(255,254,254,0.5)',backdropFilter:'blur(1px)',borderRadius:'20px'}}
+      name="normal_login"
+      className="login-form"
+      initialValues={{ remember: true }}
+      onFinish={onFinish}
+    >
+        <h1 style={{padding:'20px'}}>Signin</h1>
```
Comment (tulasikurete) [inline src/app/signin/page.tsx]: use ant theme

**Comment by saiganesharaja** [comment]
Screenshot from 2024-02-27 15-50-38

**Comment by saiganesharaja** [comment]
Screenshot from 2024-02-27 15-50-51

**Comment by saiganesharaja** [comment]
Screenshot from 2024-03-04 10-50-23

**Comment by saiganesharaja** [comment]
Screenshot from 2024-03-04 10-52-10

**Review by tulasikurete — APPROVED** [review]

**Comment by saiganesharaja** [comment]
Screenshot from 2024-03-05 11-36-54

**Review by NagaDurga16 — APPROVED** [review]

**Review by venkatatejadadi — APPROVED** [review]

**Review by gandham246 — APPROVED** [review]

**Review by Namahsivaya7 — APPROVED** [review]

**Pull request merged**